client/src/scss/gallery.scss

```diff
@@ -69,6 +69,7 @@
   max-height: 200px;
   min-height: 175px;
   padding: 10px 5px;
+  overflow-x: scroll;
```

Comment:
I think is a vertical scroll bar that Berk needs, reduce the height of your browser and you will see that the bottom left panel never gets a scroll bar, that is the problem he is seeing I think.